packs/linxImpulse/loaders/search.ts

```diff
@@ -0,0 +1,158 @@
+import type { ProductListingPage } from "deco-sites/std/commerce/types.ts";
+import type {
+  SearchParams,
+  SearchProductsResponse,
+  Sort,
+} from "deco-sites/std/packs/linxImpulse/types.ts";
+
+import { toProduct } from "deco-sites/std/packs/linxImpulse/utils/transform.ts";
+import { createClient } from "deco-sites/std/commerce/linxImpulse/client.ts";
+
+import type { Context } from "deco-sites/std/packs/linxImpulse/accounts/linxImpulse.ts";
+
+const sortOptions = [
+  { value: "relevance", "label": "Relevância" },
+  { value: "pid", "label": "Id de produto" },
+  { value: "ascPrice", "label": "Menor preço" },
+  { value: "descPrice", "label": "Maior preço" },
+  { value: "descDate", "label": "Lançamentos" },
+  { value: "ascSold", "label": "Menor venda" },
+  { value: "descSold", "label": "Maior venda" },
+  { value: "descReview", "label": "Maior avaliação" },
+  { value: "ascReview", "label": "Menor avaliação" },
+  { value: "descDiscount", "label": "Maiores descontos" },
```

Comment:
A ordenaçao aqui é importante apenas coisas que usaremos na loja entao coisas como "menos vendidos" nao é algo que importa para o usuario ou que o faria comprar mais.
```suggestion
  { value: "relevance", "label": "Relevância" },
  { value: "ascPrice", "label": "Menor preço" },
  { value: "descPrice", "label": "Maior preço" },
  { value: "descDate", "label": "Lançamentos" },
  { value: "descSold", "label": "Maior venda" },
  { value: "descReview", "label": "Maior avaliação" },
  { value: "descDiscount", "label": "Maiores descontos" },
```